d3rlpy/algos/qlearning/torch/ddpg_impl.py

```diff
@@ -157,6 +157,116 @@ def q_function_optim(self) -> Optimizer:
         return self._modules.critic_optim
 
 
+class DiscreteDDPGBaseImpl(
+    ContinuousQFunctionMixin, QLearningAlgoImplBase, metaclass=ABCMeta
+):
+    _modules: DDPGBaseModules
+    _gamma: float
+    _tau: float
+    _q_func_forwarder: ContinuousEnsembleQFunctionForwarder
+    _targ_q_func_forwarder: ContinuousEnsembleQFunctionForwarder
```

Comment:
These need to be `DiscreteEnsembnleQFunctionForwarder`.

Reply:
If you means "DiscreteEnsembleQFunctionForwarder", I think this is solved now.